SIVA/Core/Discord/Automation/Autorole.cs

```diff
@@ -8,7 +8,7 @@ public class Autorole {
         public async Task Apply(SocketGuildUser user) {
             var config = ServerConfig.Get(user.Guild);
             if (!string.IsNullOrEmpty(config.Autorole)) {
-                var targetRole = user.Guild.Roles.FirstOrDefault(r => r.Name == config.Autorole);
+                var targetRole = user.Guild.Roles.FirstOrDefault(r => r.Name.ToLower() == config.Autorole.ToLower());
```

Comment:
What if the user changes the role name after they've configured autorole?

Reply:
compatibility with v1 config

Reply:
That doesn't answer my question.